android/src/fdroid/play/listings/de-DE/short-description.txt

```diff
@@ -1 +1 @@
-Open-source, community-driven maps for travelers, tourists, cyclists & hikers
+Open-Source, von der Community erstellte Karten für Reisende, Touristen, Radfahrer und Wanderer
```

Comment:
"Community" could be replaced by "Gemeinschaft" but leaving it like that should be fine too.

Reply:
I took "community" because its shorter, but it seams that its still too long (95 from 80). I need to shorten it a bit later.

Reply:
Updated with https://github.com/organicmaps/organicmaps/pull/3351/commits/d0556b2e0447f8b7777ded0ea5a376adb436e2c3

Reply:
If "Radfahrer" is shortened to "Radler", "gemeinschaftlich" could replace "gemeinsam". The meaning would be closer to "community".

"Open-Source, gemeinschaftlich erstellte Karten für Reisende, Radler und Wanderer"

It would also be possible to replace "Open-Source" with "Quelloffen". While the term "Open-Source" seems to be more common among developers, I think the term "Quelloffen" is more common among average people.

Reply:
I would still stick with "Open-Source" rather than "Quelloffen". But I really like:

> "Open-Source, gemeinschaftlich erstellte Karten für Reisende, Radler und Wanderer"

This reads much more smoothly!